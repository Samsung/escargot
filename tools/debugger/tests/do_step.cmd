step
step
next
next
s
n
c
