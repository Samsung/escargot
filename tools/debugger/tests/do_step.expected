Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_step.js:37
(escargot-debugger) step
Stopped at tools/debugger/tests/do_step.js:27 (in f1() at line:20, col:1)
(escargot-debugger) step
Stopped at tools/debugger/tests/do_step.js:24 (in g() at line:22, col:3)
(escargot-debugger) next
Stopped at tools/debugger/tests/do_step.js:28 (in f1() at line:20, col:1)
(escargot-debugger) next
Stopped at tools/debugger/tests/do_step.js:29 (in f1() at line:20, col:1)
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step.js:38
(escargot-debugger) n
Connection closed.
